.golangci.yaml

```diff
@@ -109,30 +109,37 @@ linters:
     - whitespace
     - wsl
   disable:
+  # Keep this list sort and grouped by reason
+  # You can get the list by running the command
+  # golangci-lint linters | awk 'f;/Disabled by your/{f=1}' | cut -d":" -f1 | grep -v deprecated | cut -d" " -f1 | sort
```

Comment:
This one liner is to complicated to be helpful. What are the too groups we want to separate?

Seems simpler just to sort the entire list and add a comment for some linters.

Reply:
I felt this is easy to parse the configuration file in one look. Interspersed list is complicated to look at.

Reply:
Adding the complex commands and creating 2 groups of items in the same list is very messy. The current form having comment per line looks better to me.